docs/Changelog.rst

```diff
@@ -4,6 +4,9 @@ Changelog
 ==========
 
 
+05-08-2020 ``v2``
+    We set max lenght limit for `reference` and `poNumber` fields to 35 in create order API.
```

Comment:
Typo in length

Reply:
Maybe you could be a bit more informative on what 35 is? Is the limit 35 beers? :P I'd suggest `35 characters`

Reply:
25 kratten and kassbrooden